src/component/common/Table/cells/TimeAgoCell/TimeAgoCell.tsx

```diff
@@ -0,0 +1,50 @@
+import { Tooltip, Typography } from '@mui/material';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { useLocationSettings } from 'hooks/useLocationSettings';
+import { VFC } from 'react';
+import { formatDateYMD } from 'utils/formatDate';
+import { TextCell } from '../TextCell/TextCell';
+import TimeAgo from 'react-timeago';
+
+interface ITimeAgoCellProps {
+    value?: string | number | Date;
+    emptyText?: string;
+    children?: string | number | Date;
+}
+
+export const TimeAgoCell: VFC<ITimeAgoCellProps> = ({
+    value,
+    emptyText,
+    children,
+}) => {
+    const { locationSettings } = useLocationSettings();
+
+    const date = value ?? children;
+
+    if (!date) return <TextCell>{emptyText}</TextCell>;
+
+    return (
+        <TextCell>
+            <ConditionallyRender
+                condition={Boolean(value || children)}
+                show={
+                    <Tooltip
+                        title={`Last login: ${formatDateYMD(
+                            date,
+                            locationSettings.locale
+                        )}`}
+                        arrow
+                    >
+                        <Typography noWrap variant="body2" data-loading>
+                            <TimeAgo
+                                date={new Date(date)}
+                                live={false}
+                                title={''}
```

Comment:
Can these two be removed?

Reply:
They can, but I don't think they should. 
`live={false}` is our desired behaviour for the most part, but can be a prop. Its default is `true`. 
If we don't set `title={''}` however, we get this UI behaviour where the title shows up along with our tooltip:
![image](https://user-images.githubusercontent.com/14320932/171008451-930f1ae3-a141-441a-9a95-d06666e791b8.png)